### PR TITLE
create single campaign summary page

### LINKED ISSUE #204: single campaign summary page
image

### PR DESCRIPTION
**Pull request by nehahirve**
Fixes #204
Screenshot 2021-06-23 at 14 37 30

### CODE REVIEW

**Comment by richardolsson** [comment]
Is this ready for review, @nehahirve?

**Comment by nehahirve** [comment]
yes @richardolsson

File: src/types/zetkin.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export interface ZetkinCampaign {
     title: string;
     id: number;
     organization?: ZetkinOrganization;
+    manager: string | null;
```
Comment (richardolsson) [inline src/types/zetkin.ts:7]: I'm pretty sure this wouldn't be a `string`, but an object like below, but we can fix that once it actually exists.

```ts
manage: null | {
  id: number,
  name: string,
}
```

**Review by richardolsson — APPROVED** [review]
Looking good!

**Pull request merged**